packages/@aws-cdk/aws-codedeploy/lib/server/application.ts

```diff
@@ -11,42 +13,16 @@ import { CfnApplication } from './codedeploy.generated';
  * or one defined in a different CDK Stack,
  * use the {@link #import} method.
  */
-export interface IServerApplication extends cdk.IConstruct {
+export interface IServerApplication {
```

Comment:
Why remove `extends cdk.IConstruct` here...?

Reply:
A mistake during the refactor, nice catch.